deployment/terraform/cdn.tf

```diff
@@ -17,16 +16,56 @@ resource "aws_cloudfront_distribution" "tilegarden" {
     }
   }
 
+  origin {
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      # S3 websites don't support TLS :/
```

Comment:
Is there an established way around this? It seems icky to send the second leg of the request over plaintext.

Reply:
Not if we're using the S3 website endpoint. I haven't looked at the other parts of the request path, but has there been consideration to hitting the dynamic endpoint first?

Reply:
Since we're just hitting static content, I think going over HTTP for this leg is OK.